### PR TITLE
Pre and Post observation scripts

### PR DESCRIPTION
**Pull request by drid**
Added functionality to execute shell script before and after the observation.
Usefull for turning USB devices on/off etc

### CODE REVIEW

File: satnogsclient/observer/worker.py [hunk 1]
```diff
@@ -50,6 +52,8 @@ def __init__(self, ip, port, time_to_stop=None, frequency=None, proc=None):
             self._observation_end = time_to_stop
         if proc:
             self._gnu_proc = proc
+        if _post_exec_script:
```
Comment (acinonyx) [inline satnogsclient/observer/worker.py]: If `_post_exec_script` is anything that evaluates to `False`, the assignment won't take place.

**Review by acinonyx — CHANGES_REQUESTED** [review]
Could you please rebase so that the merge commit is removed?

**Comment by drid** [comment]
Branch is based on 0.3 tag, need to rebase on master?

**Comment by ppapadeas** [comment]
@drid probably you want this as a PR to "0" branch?

**Comment by drid** [comment]
Rebased on 0

**Pull request closed without merging**